rod_cutting_problem/RodCutting.java

```diff
@@ -0,0 +1,26 @@
+/**
+Problem Statement: Given a rod of length n inches and an array of prices that contains prices of all pieces of size smaller than n. 
+Determine the maximum value obtainable by cutting up the rod and selling the pieces.
+
+Time Complexity: O(n^2)
+Space Complexity: O(n)
+*/
+
+public class RodCutting{
+
+    public static int cutRod(int[] price) {
+       	int n = price.length;
```

Comment:
@Prakash2403 it happens a lot with me too. Just remove all the spaces first, and insert them again.
This usually happens when we hit enter, it assumes tabspace for that part and we add space for our requirements.
Also make sure you remove the spaces for lines below too, use space for all of them.